packages/commonwealth/client/scripts/state/api/members/upgradeRoles.ts

```diff
@@ -0,0 +1,35 @@
+import { notifyError, notifySuccess } from 'controllers/app/notifications';
+import $ from 'jquery';
+import app from 'state';
+import type RoleInfo from '../../../models/RoleInfo';
+
+type UpgradeRolesProps = {
+  onRoleUpdate: (oldRole: RoleInfo, newRole: RoleInfo) => void;
```

Comment:
same as [this](https://github.com/hicommonwealth/commonwealth/pull/7006/files#r1525000463)